src/seed.ts

```diff
@@ -1,4 +1,312 @@
+import { removeAdjacentChars } from "skynet-mysky-utils";
+import { hash } from "tweetnacl";
+
+import { dictionary } from "../src/dictionary";
+
 export const SEED_LENGTH = 16;
 export const SEED_WORDS_LENGTH = 13;
 export const CHECKSUM_WORDS_LENGTH = 2;
 export const PHRASE_LENGTH = SEED_WORDS_LENGTH + CHECKSUM_WORDS_LENGTH;
+
+/**
+ * Generates a 15-word seed phrase for 16 bytes of entropy plus 20 bits of checksum. The dictionary length is 1024 which gives 10 bits of entropy per word.
+ *
+ * @returns - The generated phrase.
+ */
+export function generatePhrase(): string {
```

Comment:
All of these functions are mere copy-pastes right? No need to review them in detail?

Reply:
Some are brand new and some were just moved here. This one e.g. is an old function. You can look at the commits sequentially because I think I moved them only in the last commit. The new functions are a lot of bitwise math so may be annoying to review, but they are mostly the inverse of exissting functions. However I won't merge unless I'm sure you're satisfied with the new additions.